v2.6/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -1,143 +1,103 @@
 ---
-title: Going Beyond NetworkPolicy with Calico
+title: Going Beyond `NetworkPolicy` with Calico
```

Comment:
nit: we have reports that the backticks do not work in titles so it would actually be better if you could delete them. I need to update the style guide around this.